VCFv4.3.tex

```diff
@@ -503,41 +503,100 @@ \subsection{VCF tag naming conventions}
-  \item INS: Insertion of novel sequence relative to the reference
-  \item DUP: Region of elevated copy number relative to the reference
-  \item INV: Inversion of reference sequence
-  \item CNV: Copy number variable region (may be both deletion and duplication)
+\item [\null] INS:  Insertion relative to the reference
+	\begin{itemize}
+	\item [\null] MEI:  Mobile element insertion relative to the reference
+		\begin{itemize}
+		\item [\null] ALU:  Alu mobile element insertion relative to the reference
+		\item [\null] LINE1:  LINE1 mobile element insertion relative to the reference
+		\item [\null] SVA:  SVA mobile element insertion relative to the reference
+		\item [\null] HERV:  HERV mobile element insertion relative to the reference
+		\end{itemize}
+	\item [\null] RET:  Retrotransposon insertion relative to the reference
+	\end{itemize}
+\item [\null] DEL:  Deletion relative to the reference
+	\begin{itemize}
+	\item [\null] -MEI:  Absence of a mobile element present in the reference
+		\begin{itemize}
+		\item [\null] -ALU:  Absence of a Alu mobile element present in the reference
+		\item [\null] -LINE1:  Absence of a LINE1 mobile element present in the reference
+		\item [\null] -SVA:  Absence of a SVA mobile element present in the reference
+		\item [\null] -HERV:  Absence of a HERV mobile element present in the reference
+		\end{itemize}
+	\item [\null] -RET:  Absence of a retrotransposon present in the reference
+	\end{itemize}
+\item [\null] TRA:  Translocation
+\item [\null] STR:  Short tandem repeat variation
+\item [\null] INV:  Inversion relative to the reference
+\item [\null] DUP:  Duplication
+	\begin{itemize}
+	\item [\null] TANDUP:  Tandem duplication (one of Direct, Inverted-upstream, or Inverted-downstream)
+	\item [\null] DISDUP:  Dispersed duplication
+	\end{itemize}
+\item [\null] DELINS:  Deletion-insertion (replacement of reference sequence with sequence of different content or length)
+\item [\null] OTHER:  Complex variant not covered elsewhere
+	\begin{itemize}
+	\item [\null] DEL\_DUP:  Deletion + Duplication
+	\item [\null] DEL\_INV:  Deletion + Inversion
+	\item [\null] DUP\_INV:  Duplication + Inversion
+	\item [\null] DEL\_DUP\_INV:  Deletion + Duplication + Inversion
+	\end{itemize}
+\item [\null] CNV:  Copy number variation\vspace{0.3cm}
 \end{itemize}
-
+TRA (Translocation):  A VCF translocation record describes the juxtaposition of two DNA segments from disparate genomic locations (from the same or from different chromosomes). It must include two genomic coordinates, each accompanied by an orientation (see STRAND below). A VCF translocation record only describes the immediate molecular context of the novel adjacency generated by the translocation event. It should not be confused with the traditional cytogenetic notion of translocation, which is understood to imply that the remainder of the chromosome arms beyond the specified breakpoints are also involved.\vspace{0.3cm}\\
```

Comment:
This is already covered by the breakend notation. I strongly disagree with introducing yet another notation for arbitrary genomic rearrangements. BND should be used.